src/components/ParcelDetailsPopup.vue

```diff
@@ -18,7 +24,11 @@
         <tbody>
           <tr v-for="(feature, year) in history" :key="year" :class="{'bio': Boolean(feature.bio) === true, 'non-bio': Boolean(feature.bio) === false}">
```

Comment:
`<tr v-for="(feature, year) in history" :key="year" :class="{'bio': feature.bio === '1', 'non-bio': feature.bio === '0'">`

Boolean('0') renvoie true, donc toutes les cultures sont considérées comme 'bio' dans la popup :/

Reply:
Il faudrait soit faire le check avec les strings '1' et '0', soit transformer directement la property 'bio' en booléen (peut être même à la source des données)

Reply:
Bien vu, j'ai nettoyé la donnée dans la fonction `history ()`.